projects/igniteui-angular/src/lib/grids/grid/grid-filtering-advanced.spec.ts

```diff
@@ -2662,6 +2664,35 @@ describe('IgxGrid - Advanced Filtering', () => {
             }));
         });
     });
+
+    describe('', () => {
+        let fix, grid: IgxGridComponent;
+        beforeEach(fakeAsync(() => {
+            resizeObserverIgnoreError();
```

Comment:
There is no need to call `resizeObserverIgnoreError()` anymore. It is called in `configureTestSuite`.

Reply:
Done.